### ISSUE TITLE
Vampiric brand for race doesn't work

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#33]
tested vampire race today and vampirism doesn't work it seems

```
name:Vampire
stats:2:2:-1:2:0
skill-disarm-phys:-2
skill-disarm-magic:-2
skill-device:3
skill-save:20
skill-stealth:2
skill-search:-10
skill-melee:10
skill-shoot:-10
skill-dig:0
info:11:270:15
history:100:11:4
height:66:1:62:1
weight:150:5:120:5
obj-flag:1:PROT_FEAR
obj-flag:5:FEATHER
obj-flag:10:HOLD_LIFE
value:1:RES_LIGHT[-1]
value:15:RES_DARK[1]
value:25:RES_POIS[1]
value:35:RES_NETHER[1]
value:45:RES_TIME[1]
brand:1:VAMPIRIC
```

**Comment by draconisPW** [comment]
Indeed... *facepalm*

**Comment by draconisPW** [comment]
Fixed in 0806fe1.
